diff --git a/RedGateDatabaseInfo.xml b/RedGateDatabaseInfo.xml
new file mode 100644
index 0000000..04ba5a5
--- /dev/null
+++ b/RedGateDatabaseInfo.xml
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DatabaseInformation Version="2">
+  <ScriptFileEncoding>UTF8</ScriptFileEncoding>
+  <DefaultCollation>Latin1_General_CI_AS</DefaultCollation>
+  <DefaultSchema>dbo</DefaultSchema>
+  <DefaultUser>dbo</DefaultUser>
+  <DefaultFilegroup>PRIMARY</DefaultFilegroup>
+  <DatabaseVersion>13</DatabaseVersion>
+  <IsAzure>False</IsAzure>
+  <MaxDataFileSize>10485760</MaxDataFileSize>
+  <WriteToFileOptions>
+    <Prefixes>
+      <None>
+      </None>
+      <Table>Tables</Table>
+      <StoredProcedure>Stored Procedures</StoredProcedure>
+      <View>Views</View>
+      <Default>Defaults</Default>
+      <FullTextCatalog>Storage\Full Text Catalogs</FullTextCatalog>
+      <Function>Functions</Function>
+      <Role>Security\Roles</Role>
+      <Rule>Rules</Rule>
+      <User>Security\Users</User>
+      <UserDefinedType>Types\User-defined Data Types</UserDefinedType>
+      <Trigger>
+      </Trigger>
+      <DdlTrigger>Database Triggers</DdlTrigger>
+      <Assembly>Assemblies</Assembly>
+      <Synonym>Synonyms</Synonym>
+      <XmlSchemaCollection>Types\XML Schema Collections</XmlSchemaCollection>
+      <MessageType>Service Broker\Message Types</MessageType>
+      <Contract>Service Broker\Contracts</Contract>
+      <Queue>Service Broker\Queues</Queue>
+      <Service>Service Broker\Services</Service>
+      <Route>Service Broker\Routes</Route>
+      <EventNotification>Service Broker\Event Notifications</EventNotification>
+      <PartitionScheme>Storage\Partition Schemes</PartitionScheme>
+      <PartitionFunction>Storage\Partition Functions</PartitionFunction>
+      <Field>
+      </Field>
+      <Index>
+      </Index>
+      <Schema>Security\Schemas</Schema>
+      <ServiceBinding>Service Broker\Remote Service Bindings</ServiceBinding>
+      <Certificate>Security\Certificates</Certificate>
+      <SymmetricKey>Security\Symmetric Keys</SymmetricKey>
+      <AsymmetricKey>Security\Asymmetric Keys</AsymmetricKey>
+      <CheckConstraint>
+      </CheckConstraint>
+      <FullTextStoplist>Storage\Full Text Stoplists</FullTextStoplist>
+      <ExtendedProperty>Extended Properties</ExtendedProperty>
+      <Data>Data</Data>
+      <Sequence>Sequences</Sequence>
+      <SearchPropertyList>Search Property Lists</SearchPropertyList>
+      <SecurityPolicy>Security Policies</SecurityPolicy>
+    </Prefixes>
+    <DataWriteAllFilesInOneDirectory>True</DataWriteAllFilesInOneDirectory>
+  </WriteToFileOptions>
+  <DataFileSet>
+    <Count>0</Count>
+  </DataFileSet>
+</DatabaseInformation>
\ No newline at end of file
diff --git a/Security/Schemas/info.sql b/Security/Schemas/info.sql
new file mode 100644
index 0000000..b6e7a0f
--- /dev/null
+++ b/Security/Schemas/info.sql
@@ -0,0 +1,3 @@
+CREATE SCHEMA [info]
+AUTHORIZATION [dbo]
+GO
diff --git a/Tables/dbo.ClientDatabaseLookup.sql b/Tables/dbo.ClientDatabaseLookup.sql
new file mode 100644
index 0000000..283aa46
--- /dev/null
+++ b/Tables/dbo.ClientDatabaseLookup.sql
@@ -0,0 +1,17 @@
+CREATE TABLE [dbo].[ClientDatabaseLookup]
+(
+[ClientInstanceLookup] [int] NOT NULL IDENTITY(1, 1),
+[ClientID] [int] NOT NULL,
+[DatabaseID] [int] NOT NULL,
+[InstanceID] [int] NULL,
+[Notes] [nvarchar] (250) COLLATE Latin1_General_CI_AS NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[ClientDatabaseLookup] ADD CONSTRAINT [PK_ClientInstanceLookup] PRIMARY KEY CLUSTERED  ([ClientInstanceLookup]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[ClientDatabaseLookup] ADD CONSTRAINT [FK_ClientDatabaseLookup_Clients] FOREIGN KEY ([ClientID]) REFERENCES [dbo].[Clients] ([ClientID])
+GO
+ALTER TABLE [dbo].[ClientDatabaseLookup] ADD CONSTRAINT [FK_ClientDatabaseLookup_Databases] FOREIGN KEY ([DatabaseID]) REFERENCES [info].[Databases] ([DatabaseID])
+GO
+ALTER TABLE [dbo].[ClientDatabaseLookup] ADD CONSTRAINT [FK_ClientDatabaseLookup_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/dbo.Clients.sql b/Tables/dbo.Clients.sql
new file mode 100644
index 0000000..8e62be3
--- /dev/null
+++ b/Tables/dbo.Clients.sql
@@ -0,0 +1,9 @@
+CREATE TABLE [dbo].[Clients]
+(
+[ClientID] [int] NOT NULL IDENTITY(1, 1),
+[ClientName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL,
+[External] [bit] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[Clients] ADD CONSTRAINT [PK_dbo.Clients] PRIMARY KEY CLUSTERED  ([ClientID]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.InstanceList.sql b/Tables/dbo.InstanceList.sql
new file mode 100644
index 0000000..74de72f
--- /dev/null
+++ b/Tables/dbo.InstanceList.sql
@@ -0,0 +1,16 @@
+CREATE TABLE [dbo].[InstanceList]
+(
+[InstanceID] [int] NOT NULL IDENTITY(1, 1),
+[ComputerName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL,
+[ServerName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL,
+[InstanceName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL,
+[isClustered] [bit] NOT NULL,
+[Port] [int] NOT NULL,
+[Inactive] [bit] NULL CONSTRAINT [DF_InstanceList_Inactive] DEFAULT ((0)),
+[Environment] [nvarchar] (25) COLLATE Latin1_General_CI_AS NULL,
+[Location] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[NotContactable] [bit] NULL CONSTRAINT [DF_InstanceList_NotContactable] DEFAULT ((0))
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[InstanceList] ADD CONSTRAINT [PK_InstanceList_ID] PRIMARY KEY CLUSTERED  ([InstanceID]) ON [PRIMARY]
+GO
diff --git a/Tables/info.AgentJobDetail.sql b/Tables/info.AgentJobDetail.sql
new file mode 100644
index 0000000..c35100e
--- /dev/null
+++ b/Tables/info.AgentJobDetail.sql
@@ -0,0 +1,18 @@
+CREATE TABLE [info].[AgentJobDetail]
+(
+[AgentJobDetailID] [int] NOT NULL IDENTITY(1, 1),
+[Date] [datetime] NOT NULL,
+[InstanceID] [int] NOT NULL,
+[Category] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL,
+[JobName] [nvarchar] (250) COLLATE Latin1_General_CI_AS NOT NULL,
+[Description] [nvarchar] (750) COLLATE Latin1_General_CI_AS NOT NULL,
+[IsEnabled] [bit] NOT NULL,
+[Status] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[LastRunTime] [datetime] NULL,
+[Outcome] [nvarchar] (50) COLLATE Latin1_General_CI_AS NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[AgentJobDetail] ADD CONSTRAINT [PK_info.AgentJobDetail] PRIMARY KEY CLUSTERED  ([AgentJobDetailID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[AgentJobDetail] ADD CONSTRAINT [FK_info.AgentJobDetail_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.AgentJobServer.sql b/Tables/info.AgentJobServer.sql
new file mode 100644
index 0000000..49565e3
--- /dev/null
+++ b/Tables/info.AgentJobServer.sql
@@ -0,0 +1,16 @@
+CREATE TABLE [info].[AgentJobServer]
+(
+[AgentJobServerID] [int] NOT NULL IDENTITY(1, 1),
+[Date] [datetime] NOT NULL,
+[InstanceID] [int] NOT NULL,
+[NumberOfJobs] [int] NOT NULL,
+[SuccessfulJobs] [int] NOT NULL,
+[FailedJobs] [int] NOT NULL,
+[DisabledJobs] [int] NOT NULL,
+[UnknownJobs] [int] NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[AgentJobServer] ADD CONSTRAINT [PK_Info.AgentJobServer] PRIMARY KEY CLUSTERED  ([AgentJobServerID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[AgentJobServer] ADD CONSTRAINT [FK_Info.AgentJobServer_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.Databases.sql b/Tables/info.Databases.sql
new file mode 100644
index 0000000..8d9f377
--- /dev/null
+++ b/Tables/info.Databases.sql
@@ -0,0 +1,51 @@
+CREATE TABLE [info].[Databases]
+(
+[DatabaseID] [int] NOT NULL IDENTITY(1, 1),
+[InstanceID] [int] NOT NULL,
+[Name] [nvarchar] (256) COLLATE Latin1_General_CI_AS NULL,
+[DateAdded] [datetime2] NULL,
+[DateChecked] [datetime2] NULL,
+[AutoClose] [bit] NULL,
+[AutoCreateStatisticsEnabled] [bit] NULL,
+[AutoShrink] [bit] NULL,
+[AutoUpdateStatisticsEnabled] [bit] NULL,
+[AvailabilityDatabaseSynchronizationState] [nvarchar] (16) COLLATE Latin1_General_CI_AS NULL,
+[AvailabilityGroupName] [nvarchar] (128) COLLATE Latin1_General_CI_AS NULL,
+[CaseSensitive] [bit] NULL,
+[Collation] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[CompatibilityLevel] [nvarchar] (15) COLLATE Latin1_General_CI_AS NULL,
+[CreateDate] [datetime2] NULL,
+[DataSpaceUsageKB] [float] NULL,
+[EncryptionEnabled] [bit] NULL,
+[IndexSpaceUsageKB] [float] NULL,
+[IsAccessible] [bit] NULL,
+[IsFullTextEnabled] [bit] NULL,
+[IsMirroringEnabled] [bit] NULL,
+[IsParameterizationForced] [bit] NULL,
+[IsReadCommittedSnapshotOn] [bit] NULL,
+[IsSystemObject] [bit] NULL,
+[IsUpdateable] [bit] NULL,
+[LastBackupDate] [datetime2] NULL,
+[LastDifferentialBackupDate] [datetime2] NULL,
+[LastLogBackupDate] [datetime2] NULL,
+[Owner] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[PageVerify] [nvarchar] (17) COLLATE Latin1_General_CI_AS NULL,
+[ReadOnly] [bit] NULL,
+[RecoveryModel] [nvarchar] (10) COLLATE Latin1_General_CI_AS NULL,
+[ReplicationOptions] [nvarchar] (40) COLLATE Latin1_General_CI_AS NULL,
+[SizeMB] [float] NULL,
+[SnapshotIsolationState] [nvarchar] (10) COLLATE Latin1_General_CI_AS NULL,
+[SpaceAvailableKB] [float] NULL,
+[Status] [nvarchar] (35) COLLATE Latin1_General_CI_AS NULL,
+[TargetRecoveryTime] [int] NULL,
+[InActive] [bit] NULL,
+[LastRead] [datetime2] NULL,
+[LastWrite] [datetime2] NULL,
+[LastReboot] [datetime2] NULL,
+[LastDBCCDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[Databases] ADD CONSTRAINT [PK_Databases] PRIMARY KEY CLUSTERED  ([DatabaseID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[Databases] ADD CONSTRAINT [FK_Databases_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.DiskSpace.sql b/Tables/info.DiskSpace.sql
new file mode 100644
index 0000000..b034142
--- /dev/null
+++ b/Tables/info.DiskSpace.sql
@@ -0,0 +1,16 @@
+CREATE TABLE [info].[DiskSpace]
+(
+[DiskSpaceID] [int] NOT NULL IDENTITY(1, 1),
+[Date] [datetime] NOT NULL,
+[InstanceID] [int] NOT NULL,
+[DiskName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[Label] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[Capacity] [decimal] (7, 2) NULL,
+[FreeSpace] [decimal] (7, 2) NULL,
+[Percentage] [int] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[DiskSpace] ADD CONSTRAINT [PK_DiskSpace_1] PRIMARY KEY CLUSTERED  ([DiskSpaceID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[DiskSpace] ADD CONSTRAINT [FK_DiskSpace_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.SQLInfo.sql b/Tables/info.SQLInfo.sql
new file mode 100644
index 0000000..1a28a3e
--- /dev/null
+++ b/Tables/info.SQLInfo.sql
@@ -0,0 +1,60 @@
+CREATE TABLE [info].[SQLInfo]
+(
+[SQLInfoID] [int] NOT NULL IDENTITY(1, 1),
+[DateChecked] [datetime] NULL,
+[DateAdded] [datetime] NULL,
+[ServerName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[InstanceName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[SQLVersionString] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[SQLVersion] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[ServicePack] [nvarchar] (3) COLLATE Latin1_General_CI_AS NULL,
+[Edition] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[ServerType] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[Collation] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[IsHADREnabled] [bit] NULL,
+[SQLServiceAccount] [nvarchar] (35) COLLATE Latin1_General_CI_AS NULL,
+[SQLService] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[SQLServiceStartMode] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[BAckupDirectory] [nvarchar] (256) COLLATE Latin1_General_CI_AS NULL,
+[BrowserAccount] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[BrowserStartMode] [nvarchar] (25) COLLATE Latin1_General_CI_AS NULL,
+[IsSQLClustered] [bit] NULL,
+[ClusterName] [nvarchar] (25) COLLATE Latin1_General_CI_AS NULL,
+[ClusterQuorumstate] [nvarchar] (20) COLLATE Latin1_General_CI_AS NULL,
+[ClusterQuorumType] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[C2AuditMode] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[CostThresholdForParallelism] [tinyint] NULL,
+[MaxDegreeOfParallelism] [tinyint] NULL,
+[DBMailEnabled] [bit] NULL,
+[DefaultBackupCComp] [bit] NULL,
+[FillFactor] [tinyint] NULL,
+[MaxMem] [int] NULL,
+[MinMem] [int] NULL,
+[RemoteDacEnabled] [bit] NULL,
+[XPCmdShellEnabled] [bit] NULL,
+[CommonCriteriaComplianceEnabled] [bit] NULL,
+[DefaultFile] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[DefaultLog] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[HADREndpointPort] [int] NULL,
+[ErrorLogPath] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[InstallDataDirectory] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[InstallSharedDirectory] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[IsCaseSensitive] [bit] NULL,
+[IsFullTextInstalled] [bit] NULL,
+[LinkedServer] [nvarchar] (max) COLLATE Latin1_General_CI_AS NULL,
+[LoginMode] [nvarchar] (20) COLLATE Latin1_General_CI_AS NULL,
+[MasterDBLogPath] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[MasterDBPath] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[NamedPipesEnabled] [bit] NULL,
+[OptimizeAdhocWorkloads] [bit] NULL,
+[InstanceID] [int] NULL,
+[AGListener] [nvarchar] (150) COLLATE Latin1_General_CI_AS NULL,
+[AGs] [nvarchar] (150) COLLATE Latin1_General_CI_AS NULL,
+[AGListenerPort] [nvarchar] (250) COLLATE Latin1_General_CI_AS NULL,
+[AGListenerIPs] [nvarchar] (150) COLLATE Latin1_General_CI_AS NULL
+) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
+GO
+ALTER TABLE [info].[SQLInfo] ADD CONSTRAINT [PK__SQL__50A5926BC7005F29] PRIMARY KEY CLUSTERED  ([SQLInfoID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[SQLInfo] ADD CONSTRAINT [FK_SQLInfo_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.ServerOSInfo.sql b/Tables/info.ServerOSInfo.sql
new file mode 100644
index 0000000..1482947
--- /dev/null
+++ b/Tables/info.ServerOSInfo.sql
@@ -0,0 +1,18 @@
+CREATE TABLE [info].[ServerOSInfo]
+(
+[ServerOSInfoID] [int] NOT NULL IDENTITY(1, 1),
+[DateChecked] [datetime] NULL,
+[ServerName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[DNSHostName] [nvarchar] (50) COLLATE Latin1_General_CI_AS NULL,
+[Domain] [nvarchar] (30) COLLATE Latin1_General_CI_AS NULL,
+[OperatingSystem] [nvarchar] (100) COLLATE Latin1_General_CI_AS NULL,
+[NoProcessors] [tinyint] NULL,
+[IPAddress] [nvarchar] (15) COLLATE Latin1_General_CI_AS NULL,
+[RAM] [int] NULL,
+[InstanceID] [int] NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[ServerOSInfo] ADD CONSTRAINT [PK__ServerOS__50A5926BC7005F29] PRIMARY KEY CLUSTERED  ([ServerOSInfoID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[ServerOSInfo] ADD CONSTRAINT [FK_ServerOSInfo_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
diff --git a/Tables/info.SuspectPages.sql b/Tables/info.SuspectPages.sql
new file mode 100644
index 0000000..6bc3043
--- /dev/null
+++ b/Tables/info.SuspectPages.sql
@@ -0,0 +1,19 @@
+CREATE TABLE [info].[SuspectPages]
+(
+[SuspectPageID] [int] NOT NULL IDENTITY(1, 1),
+[DatabaseID] [int] NOT NULL,
+[DateChecked] [datetime] NOT NULL,
+[FileName] [varchar] (2000) COLLATE Latin1_General_CI_AS NOT NULL,
+[Page_id] [bigint] NOT NULL,
+[EventType] [nvarchar] (24) COLLATE Latin1_General_CI_AS NOT NULL,
+[Error_count] [int] NOT NULL,
+[last_update_date] [datetime] NOT NULL,
+[InstanceID] [int] NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [info].[SuspectPages] ADD CONSTRAINT [PK_SuspectPages] PRIMARY KEY CLUSTERED  ([SuspectPageID]) ON [PRIMARY]
+GO
+ALTER TABLE [info].[SuspectPages] ADD CONSTRAINT [FK_SuspectPages_Databases] FOREIGN KEY ([DatabaseID]) REFERENCES [info].[Databases] ([DatabaseID])
+GO
+ALTER TABLE [info].[SuspectPages] ADD CONSTRAINT [FK_SuspectPages_InstanceList] FOREIGN KEY ([InstanceID]) REFERENCES [dbo].[InstanceList] ([InstanceID]) ON DELETE CASCADE
+GO
